external/downstream/v2rdm_casscf/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ if(${ENABLE_v2rdm_casscf})
 
         ExternalProject_Add(v2rdm_casscf_external
             DEPENDS psi4-core
-            URL https://github.com/loriab/v2rdm_casscf/archive/929d7fc.tar.gz
+            URL https://github.com/loriab/v2rdm_casscf/archive/v2rdm8.tar.gz
```

Comment:
What's the reason for this change? It does not appear to be related to docc/socc.

Put it in a separate pull.

Reply:
No, this is a change that I've discussed with Lori. This is an interim change for v2rdm tests not to fail.

Please ask for clarification before making demands.

Reply:
> No, this is a change that I've discussed with Lori. This is an interim change for v2rdm tests not to fail.
> 
> Please ask for clarification before making demands.

See above: "What's the reason for this change? It does not appear to be related to docc/socc."

Reply:
Current V2RDM uses `doccpi_` and `soccpi_`. This changes the copy of V2RDM we use for test purposes to a branch of Lori's where those have been replaced with the new function calls.

Reply:
Because the wfn interface changed, the v2rdm plugin needed changes. there were some that arose from this PR and other from another one. last two commits of https://github.com/loriab/v2rdm_casscf/commits/v2rdm8 . the change above is loosening from a particular commit that was the branch head to the branch itself.